cargo-dist/src/cli.rs

```diff
@@ -101,18 +100,18 @@ pub struct Cli {
     #[clap(long)]
     #[clap(help_heading = "GLOBAL OPTIONS", global = true)]
     pub tag: Option<String>,
-
-    // Add the args from the "real" build command
-    #[clap(flatten)]
-    pub build_args: BuildArgs,
 }
 
 #[derive(Subcommand)]
 pub enum Commands {
     /// Build artifacts
     #[clap(disable_version_flag = true)]
     Build(BuildArgs),
-    /// Initialize default settings in your Cargo.toml
+    /// Setup or update cargo-dist
+    ///
+    /// This will interactively guide you through the process of selecting configuration options
+    /// and will also automatically run 'cargo dist generate-ci' afterwards as necessary. It will
```

Comment:
interesting choice to auto run it! (not sure how i feel but it's not immediately negative heh). we may want to emphasize that some how (is there out put that says it? if so that's likely enough)

Reply:
related q: can running the generate-ci be destructive at all?

Reply:
also note that this behaviour was "totally" non-destructive in the original design of `init` where it refused to run if you were already init